docs/sample_config/default_config.nu

```diff
@@ -219,6 +219,7 @@ let-env config = {
   disable_table_indexes: false # set to true to remove the index column from tables
   cd_with_abbreviations: false # set to true to allow you to do things like cd s/o/f and nushell expand it to cd some/other/folder
   case_sensitive_completions: false # set to true to enable case-sensitive completions
+  enable_external_completion: true # set to false to prevent nushell looks into $env.PATH and find more suggestions, recommended for WSL users
```

Comment:
```suggestion
  enable_external_completion: true # set to false to prevent nushell looking into $env.PATH to find more suggestions. `false` recommended for WSL users as this look up may be very slow.
```

Reply:
Thanks for the comment!